pyiron_base/interfaces/has_hdf.py

```diff
@@ -106,6 +106,35 @@ class HasHDF(ABC):
     >>> hdf.remove_file()
     >>> pr.remove(enable=True)
 
+    When using this class as a mixin that also derive from classes that have a
+    legacy implementation here's a simple recipe
+
+    >>> class MyOldClass:
+    ...     def to_hdf(self, hdf, group_name):
+    ...         pass # snip
```

Comment:
I'm actually not familiar with `# snip`. Is this to imply that the old class might actually have code here?

Reply:
Ah, I was just lazy yesterday and thought "snip" is understandable as "more of the same".  I'll bulk it out later.

Reply:
Maybe it is clear, it might just be my Boomer showing 😂